src/Components/Header/Header2.jsx

```diff
@@ -1,21 +1,81 @@
-import React, {Component} from 'react';
+import React, {useState, useEffect} from 'react';
 import Menu from './Menu';
 import FlowSelector from "../FlowSelector/FlowSelector";
+import { connect } from "react-redux";
+import { push } from "connected-react-router";
 
-export default class Header2 extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {};
-    }
-
-    render() {
-        return (
-              <div className="header2-container">
-              <FlowSelector>
-                  <button>1. Letter of Interest</button>
-                  <button disabled>2. Full Application</button>
-              </FlowSelector>
-              </div>
-        );
-    }
+import { makeStyles } from "@material-ui/core/styles";
+
+const APPLICATION_STAGE = {
+  interest: "letter_of_interest",
+  full: "full_application"
+};
+
+//return current application stage from path
+function getNavId(pathname) {
+  const parts = pathname.split("/");
+  return (parts.length>=2 && parts[2] ==="full") ? APPLICATION_STAGE.full : APPLICATION_STAGE.interest;
 }
+
+const useStyles = makeStyles({
+  selected: {
+    borderColor: "#005EB8"
+  },
+  unselected: {
+    borderColor: "transparent"
+  }
+});
+
+const Header2 = ({ pathname, push }) => {
```

Comment:
Perhaps a better name would be good - what is header2?

Reply:
Good question. I think it was from a previous branch that I was on to review and branched from